_posts/2022-06-02-tf-2-9/tf-2-9.Rmd

```diff
@@ -0,0 +1,127 @@
+---
+title: "Tensorflow and Keras 2.9"
+description: > 
+  New tensorflow and keras releases bring improvements big and small.
+author:
+  - name: Tomasz Kalinowski
+    affiliation: RStudio
+    affiliation_url: https://www.rstudio.com/
+slug: kalinowskiDLwR2e
+date: 2022-06-02
+output:
+  distill::distill_article:
+    self_contained: false
+    toc: true
+categories:
+  - Tensorflow
+  - Keras
+  - Deep Learning
+  - R
+editor_options:
+  markdown:
+    wrap: 72
+preview: images/coffee-n-milk.jpg
```

Comment:
I always try to use landscape-format fotos for the preview, as this looks better on the preview page. (Unless you have a long abstract.)
Anyway the lizard is awesome!!